bitnami/moodle/4.4/debian-12/rootfs/post-init.d/sql-mysql.sh

```diff
@@ -21,7 +21,7 @@ fi
 mysql_execute() {
     local -r sql_file="${1:?missing file}"
     local failure=0
-    mysql_cmd=("mysql" "-h" "$MARIADB_HOST" "-P" "$MARIADB_PORT_NUMBER" "-u" "$MARIADB_ROOT_USER")
+    mysql_cmd=("mysql" "-h" "$MARIADB_HOST" "-P" "$MARIADB_PORT_NUMBER" "-u" "$MARIADB_ROOT_USER" "-D" "${MARIADB_DATABASE}")
```

Comment:
This change need to be added to the other moodle's versions too.

Reply:
I have now alligned the versions to use the same script